frontend/src/login/LoginFailedDialog.jsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import PropTypes from "prop-types";
+import {
+  Button,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogContentText,
+  DialogTitle,
+  makeStyles
+} from "@material-ui/core";
+
+const useStyles = makeStyles((theme) => ({
```

Comment:
There's a code smell here for `'theme' is defined but never used (no-unused-vars)`, can this be resolved by removing the keyword `theme`?